trin-execution/src/execution.rs

```diff
@@ -192,9 +201,12 @@ impl State {
             self.execution_position
                 .increase_transaction_index(self.database.db.clone())?;
             self.database.commit(evm_result.state);
```

Comment:
nit: I think it would be useful to have separate timer just for this step (`self.database.commit(_)`) as I think most of the time is spend there.

Reply:
This will be addressed in the next PR which will include a graph specific to transaction execution performance